### PR TITLE
Masterlist Page Filtered Table + Styling

### PR DESCRIPTION
**Pull request by stephen-mcneal**

### CODE REVIEW

File: dashboard/static/scripts/script.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+function toggledropdown() {
+        var dropdown = document.getElementById('myDropdown');
+        var displaySetting = dropdown.style.display;
+        if (displaySetting == 'block') {
+            dropdown.style.display = 'none';
+          }
+          else {
+            dropdown.style.display = 'block';
+          }
```
Comment (Pkoiralap) [inline dashboard/static/scripts/script.js]: indentation

File: dashboard/static/scripts/script.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    
+  }
```
Comment (Pkoiralap) [inline dashboard/static/scripts/script.js]: new line at eof

File: dashboard/templates/base.html [hunk 3]
```diff
@@ -68,4 +70,4 @@
     </div>
 </body>
 
-</html>
+</html>
```
Comment (Pkoiralap) [inline dashboard/templates/base.html]: new line at eof

File: dashboard/templates/cdi_masterlist/CDI_MASTERLIST.html [hunk 4]
```diff
@@ -1,26 +1,99 @@
...
+
+		</div>
+		</div>
+
+
+
+
+	</body>
+
+	{% endblock %}
```
Comment (Pkoiralap) [inline dashboard/templates/cdi_masterlist/CDI_MASTERLIST.html]: new line at eof

File: dashboard/views.py [hunk 5]
```diff
@@ -120,7 +120,7 @@ def get(self, request):
 
         ml_filter = MasterlistFilter(request.GET, queryset=masterlist_qs)
         masterlist = list(ml_filter.qs)
-
+
```
Comment (Pkoiralap) [inline dashboard/views.py:123]: indent

**Comment by Pkoiralap** [comment]
Since there are style changes it would be nice to have a picture of how the page looks.

**Comment by andrewdweis** [comment]
Looks great - Can you change the table to only feature the following attributes:
Title (Hyperlinked with Catalog URL), Organization, CDI Themes, Metadata Type, Status

File: dashboard/views.py (hunk 5 above)
Comment (stephen-mcneal) [reply dashboard/views.py:123]: what do you mean?

**Comment by stephen-mcneal** [comment]
Table was edited accordingly:
masterlist

**Comment by andrewdweis** [comment]
Great Updates - I had one more thing that I think would be good to include in this branch push.

Can you include the two hyperlink texts to the left of the filter (outlined in the functionality doc) for the following:
"QA Updates" to page /cdi-masterlist/qa-updates
"Download" to page /cdi-masterlist/download

**Comment by stephen-mcneal** [comment]
Updated page with "QA Updates" link to page /cdi-masterlist/qa-updates and "Download" link to page /cdi-masterlist/download
masterlist

**Comment by andrewdweis** [comment]
Looks great

**Pull request merged**